### PR TITLE
WIP - Refactorez login page

### PR DESCRIPTION
**Pull request by sebikovacs**
Please check LoginForm.js and HomePage.js, the rest are formatting changes.

### CODE REVIEW

File: magic-volunteers-front-end/src/components/LoginComponent.css [hunk 1]
```diff
@@ -1,229 +1,299 @@
...
+    margin: 0;
+    padding: 0;
+    display: block;
 }
 
 .action-selector li {
-  float: left;
-  width: 50%;
-  text-align: center;
-  font-family: "Shadows Into Light Two";
```
Comment (melaniam) [inline magic-volunteers-front-end/src/components/LoginComponent.css]: This font-family is the one used on MagiCamp site. We should go with it, to keep the same style.

File: magic-volunteers-front-end/src/components/LoginForm.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+                CredentialsUtils.storeCredentials(
+                    credentials.userName,
+                    credentials.token,
+                    cookieDuration
+                );
+
+                console.log("success");
+
+                // Test purposes.
```
Comment (melaniam) [inline magic-volunteers-front-end/src/components/LoginForm.js]: Can you please remove these commented out lines, if possible? We're not using them anyway, and they were there just for testing purposes anyway.

File: magic-volunteers-front-end/src/components/LoginComponent.css (hunk 1 above)
Comment (sebikovacs) [reply magic-volunteers-front-end/src/components/LoginComponent.css]: I don't agree since this is not a presentational site but a Dashboard.

Comment (melaniam) [reply magic-volunteers-front-end/src/components/LoginComponent.css]: it's still (supposed to be) used by around 1000 volunteers who would feel more comfortable seeing the same font as on the main site :P I don't see why it bothers you that much :D does it have such a huge impact performance wise?

Comment (sebikovacs) [reply magic-volunteers-front-end/src/components/LoginComponent.css]: no it doesn't it's just an ugly font and brings no value to a dashboard. We can use it for the main titles but I'd advise against anywhere else.

Comment (melaniam) [reply magic-volunteers-front-end/src/components/LoginComponent.css]: Only the main titles! Thanks!

**Pull request merged**